flask_rebar/compat.py

```diff
@@ -0,0 +1,60 @@
+import sys
+from distutils.version import StrictVersion
+
+import marshmallow
+
+
+PY2 = int(sys.version_info[0]) == 2
+MARSHMALLOW_VERSION = StrictVersion(marshmallow.__version__).version
+MARSHMALLOW_MAJOR_VERSION = MARSHMALLOW_VERSION[0]
```

Comment:
It looks like everywhere this is used, it is used like `MARSHMALLOW_MAJOR_VERSION == 2`. That argues for factoring this value out into its own constant, e.g. `MALLOW_V2 = (MARSHMALLOW_MAJOR_VERSION == 2)`.